src/lib.rs

```diff
@@ -151,30 +202,35 @@ struct CommonOptions {
     #[structopt(short = "O", long)]
     optimize: bool,
 
-    /// Optimization level for generated functions (0 (none), 1, 2 (most), or s
-    /// (size))
+    /// Optimization level for generated functions
+    /// Supported levels: 0 (none), 1, 2 (most), or s (size); default is "most"
     #[structopt(
         long,
+        value_name = "LEVEL",
         parse(try_from_str = parse_opt_level),
-        default_value = "2",
+        verbatim_doc_comment,
     )]
-    opt_level: wasmtime::OptLevel,
+    opt_level: Option<wasmtime::OptLevel>,
 
-    /// Other Cranelift flags to be passed down to Cranelift.
-    #[structopt(long, parse(try_from_str = parse_cranelift_flag))]
+    /// Cranelift common flags to set.
+    #[structopt(long = "cranelift-flag", value_name = "NAME=VALUE", parse(try_from_str = parse_cranelift_flag))]
     cranelift_flags: Vec<CraneliftFlag>,
 
+    /// The Cranelift ISA preset to use.
+    #[structopt(long, value_name = "PRESET")]
+    cranelift_preset: Option<String>,
```

Comment:
To bikeshed a bit, could this perhaps be called `cranelift_enable` and the above method is called `cranelift_set`? (I figure that if cranelift wants two different operations we could probably mostly just try to mirror them here well enough